specification/metadata/metadata.mdpp

```diff
@@ -0,0 +1,11 @@
+# Metadata
+
+The FOCUS specification defines a metadata structure that is to be supplied by data providers to facilitate practioners in data ingestion, data locations, schema of the FOCUS dataset.
```

Comment:
> is to be supplied

MAY, SHOULD or MUST?

I would vote for a MUST.

Reply:
```suggestion
The FOCUS specification defines a metadata structure that MUST be supplied by data providers to facilitate practitioners in data ingestion, data locations, schema of the FOCUS dataset.
```

Reply:
@rileyjenk added some of your entity conversation here. Please feel free to add more suggestions.